From 5dcca8bea45497e64ac7ec0aa6bd7f53b75df163 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Wed, 8 Jun 2016 15:45:29 +0800
Subject: [PATCH] ARM: dts: rockchip: add peripherals-req-type-burst for rk3036

As the rk3036 only support the burst mode, so we need add it since
The commit 8e770f371cc27
"dmaengine: pl330: add burst mode according to dts config"
had been supported in kernel.

Change-Id: I074aa9a98c78a8ad8b1263a07690ffc2f8dfa718
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm/boot/dts/rk3036.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk3036.dtsi b/arch/arm/boot/dts/rk3036.dtsi
index 1ed48dd829e5..008dd36ef5c4 100644
--- a/arch/arm/boot/dts/rk3036.dtsi
+++ b/arch/arm/boot/dts/rk3036.dtsi
@@ -102,6 +102,7 @@
 				     <GIC_SPI 1 IRQ_TYPE_LEVEL_HIGH>;
 			#dma-cells = <1>;
 			arm,pl330-broken-no-flushp;
+			peripherals-req-type-burst;
 			clocks = <&cru ACLK_DMAC2>;
 			clock-names = "apb_pclk";
 		};
-- 
2.35.3

